CHANGELOG.md

```diff
@@ -150,6 +150,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Fixed
 
+- We fixed an issue where a NullPointerException was thrown when userAndHost was not initialized in the preferences. The initialization now correctly falls back to getUserHostInfo() to prevent the error. [#14497](https://github.com/JabRef/jabref/issues/14497)
```

Comment:
Second sentence is an implementation detail. Maybe also shorten the first sentence, because NPE is unknown to the users. Think of a non-programmer using JabRef.